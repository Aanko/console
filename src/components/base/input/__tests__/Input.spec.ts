import { describe, expect, it } from "vitest";
import { VInput } from "../index";

describe("Input", () => {
  it("should render", () => {
    expect(VInput).toBeDefined();
  });
});
