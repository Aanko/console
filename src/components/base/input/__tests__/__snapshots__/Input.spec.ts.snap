// Vitest Snapshot v1

exports[`Input > should render 1`] = `
"<div class=\\"input-wrapper\\">
  <!--v-if--><input class=\\"input-md\\" type=\\"text\\">
  <!--v-if-->
</div>"
`;
